### DISCUSSION TITLE
能不能让用户自己指定数据库啊，sqlite3 老锁，太难了，希望增加mysql等其他的数据库支持

### DISCUSSION THREAD

**Discussion by iamckun** [jxxghp/MoviePilot#2233] [建议]
能不能让用户自己指定数据库啊，sqlite3 老锁，太难了，希望增加mysql等其他的数据库支持
![image](https://github.com/jxxghp/MoviePilot/assets/65990499/aa0fd25c-9d7b-485d-837a-6ce76a8c6797)

**Comment by cddjr** [comment]
我没这个问题，估计你使用环境有点特殊吧

**Reply by iamckun** [reply]
卡io就会这样，也没啥特殊的

**Reply by cddjr** [reply]
> 卡io就会这样，也没啥特殊的

docker的配置数据放固态去

**Comment by xiaocaiaa** [comment]
我V1没这个问题，最近V2一直锁数据库
